frr-tools/metrics/exporter.go

```diff
@@ -23,6 +23,7 @@ import (
 )
 
 var (
+	metricsHost = flag.String("host", os.Getenv("METALLB_HOST"), "metrics HTTP host address")
```

Comment:
We removed this ENV, but we are still using it. right?

Reply:
I think you are right. That's for when we still want to expose the metrics, but not securely. In that case it makes sense for the host to be the ip of a node.
Good catch again!

Reply:
I think if we remove the ENV in the `speaker.yaml`, we maybe should give it default value to "localhost". Use this instead of `os.Getenv("METALLB_HOST")`, WDYT?

Reply:
There are two ways to deploy this machinery. One is with kuberbac proxy embedded, which exposes metrics securely. In that case, we want to listen on localhost because we don't want to expose the metrics port in the wild (because the speaker is hostnetworked).
If, on the other hand, a user doesn't need kuberbac proxy, it makes sense to listen to the node ip, because then prometheus must be able to scrape directly the metrics port (which would not be possible if we listen to localhost).

This is to say that without kuberbac proxy we do need the env variable to pass in the node ip.

Reply:
Got it, this makes sense.